vyper/functions/functions.py

```diff
@@ -616,7 +616,7 @@ def as_wei_value(expr, args, kwargs, context):
         if value.value <= 0:
             raise InvalidLiteralException("Negative wei value not allowed", expr)
         sub = ['mul', value.value, denom_divisor]
-    elif value.typ.typ == 'uint256':
+    elif value.typ.typ in ['uint256', 'int128']:
```

Comment:
tuple probably cleaner
```suggestion
    elif value.typ.typ in ('uint256', 'int128'):
```

Reply:
Thanks. It's better.